src/components/LoginForm.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { Button, Grid, Link, Stack, Typography, TextField } from '@mui/material';
+const LoginInput = () => {
+  return (
+    <Grid spacing={2} container item direction="column" xs>
+      <Grid item container direction="column" spacing={5} sx={{ textAlign: 'center', alignItems: 'center' }}>
+        <Grid item xs>
+          <Typography variant="h2" fontWeight={'bold'} sx={{ marginLeft: 'auto' }}>
+            Login
+          </Typography>
+        </Grid>
+        <Grid item xs sx={{ width: '70%' }}>
+          <Stack spacing={2}>
+            <TextField type="text" label="Email" />
+            <TextField type="password" label="Password" />
+          </Stack>
```

Comment:
Following the figma, the Login text should be left-aligned, and the textfields should be the same length as the Login button. I'm assuming you coded this in with a smaller screen size since minimizing the screen makes it look fairly identical to the figma, but you need to account for different screen sizes for UX purpose

![image](https://github.com/LetsMesh/Site/assets/24869207/d170176b-54b0-435d-a793-31d54fc47dac)